packages/react-components/react-tree/library/src/hooks/useRovingTabIndexes.ts

```diff
@@ -67,6 +68,22 @@ export function useRovingTabIndex() {
     }
   }, [targetDocument, initialize]);
 
+  const hasChildrenRef = React.useRef(false);
+
+  React.useEffect(() => {
+    if (walkerRef.current) {
+      const firstChild = walkerRef.current.firstChild();
+      walkerRef.current.currentElement = walkerRef.current.root;
+      // this is an edge case where the tree is empty and the rove is not initialized
+      // we need to force update the rove to set the first child as the current element
+      // https://github.com/microsoft/fluentui/issues/34075
+      if (hasChildrenRef.current === false && firstChild) {
+        forceUpdate();
+      }
+      hasChildrenRef.current = firstChild !== null;
+    }
+  });
```

Comment:
Is it good enough? `Tree` may be disconnected from items, so there might be no update of `Tree` itself.

https://stackblitz.com/edit/vitejs-vite-hxf3fzkt

Reply:
Might not be in this case, then the solution required would be a mount/unmount hook on every treeitem, which might be too heavy on huge trees, any suggestions?